### ISSUE TITLE
VSCode: Navigate to nested function

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/TypeScript#9576]
**TypeScript Version:**  master

**Code**

```ts
function fnc1() {
    function fnc2() {}
}
```

In Visual Studio, I can use `ctrl-,` to navigate to either `fnc1` or `fnc2`.
In VSCode, I can use `ctrl-p` to navigate to `fnc1`, but not to `fnc2`.
The error also happens for functions nested in namespaces.
In both cases, I have the editors set up to point to my latest build of the master branch.

**Comment by sandersn** [comment]
This doesn't repro for me in TypeScript or in JavaScript. What version of VSCode is this?

**Comment by sandersn** [comment]
I am on version 

Version 0.10.11
Commit f291f4ad600767626b24a4b15816b04bee9a3049
Date 2016-03-11T16:19:03.130Z
Shell 0.35.6
Renderer 45.0.2454.85
Node 4.1.1

which seems ancient, so maybe it's a VSCode regression.

**Comment by andy-ms** [comment]
I updated this morning to `1.3.0`.

**Comment by DanielRosenwasser** [comment]
@andy-ms are you using an up-to-date language service?

**Comment by sandersn** [comment]
Now that I've updated to 1.3.0, I can repro this bug.

**Comment by andy-ms** [comment]
I set the `"typescript.tsdk"` settings to point to my checkout of master, so it should be completely up-to-date.

**Comment by sandersn** [comment]
Additionally, in 0.10.11, if I had a folder with files `first.ts` and `second.js`, then f1 &ndash; f4 were all found in the code below. In 1.3.0, only the outer symbol from the file *with the caret* is found. That is, `f3` is not found when `first.ts` has the caret, and `f1` is not found when `test.js` has the caret.

```ts
// first.ts
function f1() {
  function f2() {
  }
}

// second.js
function f3() {
  function f4() {
  }
}
```

**Comment by sandersn** [comment]
I am also running a checkout of master. I don't think anything else changed in my testing besides upgrading VSCode from 0.10.11 &rarr; 1.3.0.

**Comment by andy-ms** [comment]
Closed along with Microsoft/vscode#9217
